src/library_emmalloc.js

```diff
@@ -5,9 +5,8 @@
  */
 
 mergeInto(LibraryManager.library, {
-  emmalloc_unclaimed_heap_memory__deps: ['emscripten_get_sbrk_ptr'],
   emmalloc_unclaimed_heap_memory: function() {
-  	var dynamicTop = HEAPU32[_emscripten_get_sbrk_ptr()>>2];
+	var dynamicTop = _sbrk();
```

Comment:
I love the idea of just use `_sbrk()` here..  in my various attempts at this I just moved `_emscripten_get_sbrk_ptr` into normal C code (which also works), but this is simpler.